src/lib.rs

```diff
@@ -1269,6 +1274,10 @@ pub unsafe trait TryFromBytes {
         // This call may panic. If that happens, it doesn't cause any soundness
         // issues, as we have not generated any invalid state which we need to
         // fix before returning.
+        //
+        // Note that one panic condition is calling `try_into_valid` (and thus
+        // `is_bit_valid`) with a shared pointer when `Self: !NoCell`. Since
+        // `Self: NoCell`, this panic condition will not happen.
```

Comment:
This is a post-monomorphization error, not a panic, right?

Reply:
Done.